models/hashflow/ethereum/hashflow_ethereum_trades.sql

```diff
@@ -0,0 +1,424 @@
+{{ config(
+    alias = 'trades',
+    partition_by = ['block_date'],
+    materialized = 'incremental',
+    file_format = 'delta',
+    incremental_strategy = 'merge',
+    unique_key = ['block_date', 'composite_index', 'tx_hash'],
+    post_hook='{{ expose_spells(\'["ethereum"]\',
+                                "project",
+                                "hashflow",
+                                \'["justabi", "jeff-dude"]\') }}'
+    )
+}}
+
+{% set project_start_date = '2021-04-28' %}
+
+with ethereum_traces as (
+    select *
+    from {{ source('ethereum', 'traces') }}
+    where `to` in ('0x455a3B3Be6e7C8843f2b03A1cA22A5a5727ef5C4','0x9d4fc735e1a596420d24a266b7b5402fe4ec153c',
```

Comment:
Also the first address can be in lower cased

Reply:
oop - missed this! on it